rollup.config.js

```diff
@@ -3,6 +3,7 @@ import json from '@rollup/plugin-json'
 import resolve from '@rollup/plugin-node-resolve'
 import esbuild from 'rollup-plugin-esbuild'
 import minifyHtml from 'rollup-plugin-minify-html-literals'
+import nodePolyfills from 'rollup-plugin-polyfill-node'
```

Comment:
Since this includes `buffer` according to their [docs](https://github.com/FredKSchott/rollup-plugin-polyfill-node#nodejs-builtin-support-table), why are we also depend on `buffer` when doing `window.Buffer = Buffer` above?

Reply:
We only use this `nodePolyfills` plugin in umd builds, es onesgo through just esbuild. Needed to add polyfills here for some other missing "vanilla" deps.